zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -150,7 +152,7 @@ angular.module('zeppelinWebApp').controller('NotebookCtrl', function($scope, $ro
   };
 
   $scope.killSaveTimer = function() {
```

Comment:
Yeah, there is a lot of code that need refactoring.
But there is no point in adding more work to that future task.